README.md

```diff
@@ -1,3 +1,12 @@
 # quizaccess_ai
 
-A quiz accessrule subplugin that prevents users from taking a quiz containing AI features if the users do not have access to AI functionalities
+A quiz accessrule subplugin that prevents users from taking a quiz containing AI features if the users do not have access to AI functionalities.
+
+## Requirements
```

Comment:
No requirements, you made the plugin work without it.

Maybe just add a section "When do I need this plugin?" or "Purpose" where you describe what problems this plugin is solving.